src/lib.rs

```diff
@@ -681,8 +674,6 @@ impl Bump {
                 .unwrap_or_else(allocation_size_overflow);
             let layout = layout_from_size_align(size, align);
 
-            debug_assert!(size >= old_size_with_footer.unwrap_or(0) * 2);
-
```

Comment:
Not sure about this change, what do you think? Do I need to move it to the `alloc_layout_slow` method?